addons/base_import/static/src/import_model.js

```diff
@@ -426,7 +426,10 @@ export class BaseImportModel {
                     if (error.record !== undefined) {
                         this._addMessage("danger", [
                             error.rows.from === error.rows.to
-                                ? _t('Error at row %s: "%s"', error.record, error.message)
+                                ? _t('Error at row %(row)s: "%(error)s"', {
```

Comment:
Yes, we have a custom implementation of `sprintf` on the JS side, even though the only format specifier it knows is `s`. And since Odoo 17, `_t` calls `sprintf` on its arguments: https://github.com/odoo/odoo/pull/123932 😎